File: algorithms/strings/funny-string/funny-string-javascript/js/testeFunnyString.js

import {funnyString} from '../js/funnyString.js';
import assert from 'assert';

describe('funnyString', function() {

    it('A string é : Funny', function() {

        let s = 'acxz';
        let esperado = 'Funny';

        let resultado = funnyString(s)

        assert.strictEqual(resultado, esperado);
    });
});

describe('funnyString', function() {

    it('A string é : Not Funny', function() {

        let s = 'bcxz';
        let esperado = 'Not Funny';

        let resultado = funnyString(s)

        assert.strictEqual(resultado, esperado);
    });
});

describe('funnyString', function() {

    it('A string é : Not Funny', function() {

        let s = 'ivvkxq';
        let esperado = 'Not Funny';

        let resultado = funnyString(s)

        assert.strictEqual(resultado, esperado);
    });
});

File: algorithms/strings/funny-string/funny-string-java/funny-string-java/src/TesteFunnyString.java
import static org.junit.Assert.assertEquals;

import org.junit.Test;

public class TesteFunnyString {
    
    @Test
    public void testeCaso1() {

        String s = "acxz";
        String esperado = "Funny";
        
        String resultado = Resultado.funnyString(s);

        assertEquals(esperado, resultado);
    }
}
